includes/wc-stock-functions.php

```diff
@@ -403,13 +403,13 @@ function wc_get_low_stock_amount( WC_Product $product ) {
 	$low_stock_amount = $product->get_low_stock_amount();
 
 	if ( '' === $low_stock_amount && $product->is_type( 'variation' ) ) {
-		$product = wc_get_product( $product->get_parent_id() );
+		$product          = wc_get_product( $product->get_parent_id() );
 		$low_stock_amount = $product->get_low_stock_amount();
 	}
 
 	if ( '' === $low_stock_amount ) {
 		$low_stock_amount = get_option( 'woocommerce_notify_low_stock_amount', 2 );
 	}
 
-	return $low_stock_amount;
+	return intval( $low_stock_amount );
```

Comment:
Unless we're using the second argument to change the `base`, it is preferred to use `(int)` instead as it is faster than calling a type casting function.